### ISSUE TITLE
Not working on 2.1.50

### ISSUE THREAD

**Issue by AnKingMed** [hgiesel/closet#92]
I imagine this is your own fault for changing the editor :P

Do you have any timeline on updating this?

**Comment by AnKingMed** [comment]
@hgiesel I could potentially have someone help with this update. Do you need help? If so, what part do you need help with? Do you know what your timeline is?

**Comment by AnKingMed** [comment]
@hgiesel I haven't gotten any response from you on any messaging platform.  I'd really appreciate if you could let me know what your expected timeline on this is.  I want to release a youtube video encouraging people to update to 2.1.53 and explain all the new features, etc, but I'm not going to do so until all the major features work.  One major feature that isn't working is the IO one by one note type I had designed because it relies on this add-on. 

tl;dr This is urgent. Please give me an update on your timeline for releasing an update on this. I'm happy to pay people to help if needed

**Comment by closernow** [comment]
Hi, would really like to be able to see the videos you posted, and would especially like to use this plugin.
I don't know how to use this plugin yet, I can't even find an available apkg and the author's video is private.

**Comment by Ka-lip** [comment]
I just tried this addon and the shortcut does not work (ctrl + shift + C)...
I am still getting familiar with this addon but if no shortcut I think it is really difficult to  make it usable since its syntaxes are not so easy to memorize.
the version of anki is 2.1.52.
